sdk/lusid_drive/utilities/folders_api_extensions.py

```diff
@@ -102,7 +106,8 @@ def delete_folder(api_factory, drive_path):
     """
 
     if not drive_path.startswith("/"):
```

Comment:
Didn't notice this before, but maybe it would be more user friendly if the method added this leading slash instead of throwing an error?